source/_integrations/miele.markdown

```diff
@@ -186,6 +186,18 @@ Climate entities are used to control target temperatures in refrigerators, freez
 - **Robot vacuum cleaner**: Miele robot vacuum cleaners can be monitored and controlled to a limited extent. The device can be started, stopped, and paused. The fan speed can also be set.
 {% enddetails %}
 
+## Actions
+
+### Action `miele.set_program`
+
+Set and start a program for applicable appliances. Note that the device must be in a state where it will accept a new program, for example, most washing machines must be in state `On`. An error message is displayed if the device did not accept the action command.
```

Comment:
Why `On`? It being in backticks looks like it's technical, but if it was technical, it wouldn't be capitalized